src/bench/sign_transaction.cpp

```diff
@@ -62,7 +62,7 @@ static void SignTransactionSingleInput(benchmark::Bench& bench, InputType input_
     bench.minEpochIterations(100).run([&] {
         CMutableTransaction tx{unsigned_tx};
         std::map<COutPoint, Coin> coins;
-        CScript prev_spk = prev_spks[(iter++) % prev_spks.size()];
+        const CScript& prev_spk = prev_spks[(iter++) % prev_spks.size()];
```

Comment:
The `prev_spk` variable is passed to the `CTxOut` ctor, which takes a copy rather than a const reference.

This raises a concern about the correctness of the change, doesn't it?

Reply:
@hebasto are you asking because [CScript is trivially copyable](https://github.com/bitcoin/bitcoin/blob/master/src/prevector.h#L38), but https://clang.llvm.org/extra/clang-tidy/checks/performance/unnecessary-copy-initialization.html claims
> Finds local variable declarations that are initialized using the copy constructor of a non-trivially-copyable type but it would suffice to obtain a const reference.
?

Reply:
> The `prev_spk` variable is passed to the `CTxOut` ctor...

The ctor treats `CScript scriptPubKeyIn` parameter as `const`. Everything is OK.

Sorry for the noise.

Reply:
I think so.